core/ui/ControlPanel/Plugins.tid

```diff
@@ -16,4 +16,9 @@ caption: {{$:/language/ControlPanel/Plugins/Caption}}
 
 <<lingo Installed/Hint>>
 
-<$macrocall $name="tabs" tabsList="[[$:/core/ui/ControlPanel/Plugins/Installed/Plugins]] [[$:/core/ui/ControlPanel/Plugins/Installed/Themes]] [[$:/core/ui/ControlPanel/Plugins/Installed/Languages]]" default="$:/core/ui/ControlPanel/Plugins/Installed/Plugins" explicitState="$:/state/tab--86143343"/>
+<$macrocall $name="tabs"
+tabsList="[[$:/core/ui/ControlPanel/Plugins/Installed/Plugins]]
+[[$:/core/ui/ControlPanel/Plugins/Installed/Themes]]
+[[$:/core/ui/ControlPanel/Plugins/Installed/Languages]]
+[[$:/core/ui/ControlPanel/Plugins/Installed/LanguagesAddOns]]"
```

Comment:
This tab will need a new expandable "group", so users can filter for a base language. eg: "de-??"